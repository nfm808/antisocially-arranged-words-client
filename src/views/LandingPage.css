.LandingPage {
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-content: center;
  justify-content: center;
  overflow-y: hidden;
  padding-top: 2vh;
}