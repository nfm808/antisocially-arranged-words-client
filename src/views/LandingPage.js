import React from 'react'
import Hero from '../components/Hero/Hero';
import LearnMore from '../components/LearnMore/LearnMore';
import './LandingPage.css';

export default function LandingPage() {

  return (
    <section className='LandingPage'>
      {/* <Hero 
        heading='Antisocially Arranged Words'
        subHeading='Bringing the horrible coffee table game across the web.'
      />  */}
      <h1>Antisocially Arranged Words</h1>
      <h3>Dragging the horrible coffee table game across the web.</h3>
      <LearnMore />     
    </section>
  )
}
