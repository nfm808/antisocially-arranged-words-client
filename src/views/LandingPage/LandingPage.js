import React from 'react'
import CardDemo from '../../components/CardDemo/CardDemo';
import './LandingPage.css';
import Nav from '../../components/Nav/Nav';

export default function LandingPage() {

  return (
    <main className='LandingPage'>
      <h1>Antisocially Arranged Words</h1>
      <h3>Dragging the horrible coffee table game across the web.</h3>
      <Nav 
        links={[
          {to: 'create-game', name: 'Create a Game'}, 
          {to: 'rules', name: 'Rules and Such'}
        ]}  
      />
      <CardDemo />     
    </main>
  )
}
