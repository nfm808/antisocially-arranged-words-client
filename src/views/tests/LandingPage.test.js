import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from '../LandingPage';

describe('<LandingPage />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<LandingPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})