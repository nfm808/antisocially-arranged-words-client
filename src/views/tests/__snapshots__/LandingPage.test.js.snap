// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> renders without crashing 1`] = `
<Fragment>
  <Hero
    heading="Antisocially Arranged Words"
    subHeading="Bringing the horrible coffee table game across the web."
  />
  <LearnMore />
</Fragment>
`;
